docs/fleet/machines.md

```diff
@@ -92,6 +92,10 @@ For local communication between [parts](/build/configure/parts-and-remotes/#mach
 <br>
 
 You can also access the control interface using the [Viam mobile app](/fleet/#the-viam-mobile-app), which you can find on the [App Store](https://apps.apple.com/vn/app/viam-robotics/id6451424162) and on [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US).
+The Viam mobile app gives you the ability to search through the machines in your fleet, ordered by location, and control the specific machine or machine component that you need.
+You can watch the live camera feed, adjust the component or machine's runtime parameters, and switch between controllable components just as you would in the Viam app.
```

Comment:
```suggestion
The Viam mobile app gives you the ability to search through the machines in your fleet, ordered by location, and control each machine component.
You can watch any live camera feeds, adjust components' runtime parameters, and switch between controllable components just as you would in the Viam app.
```
I think it makes more sense to just talk about controlling components since there aren't really any controls for the machine as a whole. Or, you could talk about controlling the machine, in which case control of components would be implied and that'd be fine. For example, `control each machine.` instead of `control each machine component.` But I think "specific machine or machine component" implies that you can control a machine, AND/OR you can control an individual component, which could be confusing.

Reply:
I think the `search through` sounds a bit awkward.

```suggestion
The Viam mobile app gives you the ability to monitor and operate machines in your fleet.
You can control each machine component, watch any live camera feeds, adjust components' runtime parameters, and switch between controllable components just as you would in the Viam app.
```